src/components/vive-controls.js

```diff
@@ -174,6 +177,13 @@ module.exports.Component = registerComponent('vive-controls', {
     controllerObject3D.position.set(0, -0.015, 0.04);
   },
 
+  onAxisMoved: function (evt) {
+    if (evt.detail.axis[0] === 0 && evt.detail.axis[1] === 0) {
+      return;
+    }
+    this.el.emit('trackpadmoved', evt.detail);
```

Comment:
Do we want to map the axis to `x` and `y` coordinates for better ergonomics? Something like:
````javascript
this.el.emit('trackpadmoved', { x: evt,detail.axis[0], y: evt.detail.axis[1] });
````